plugins/decor/deco-layout.cpp

```diff
@@ -230,7 +231,20 @@ decoration_layout_t::action_response_t decoration_layout_t::handle_press_event(
         auto area = find_area_at(current_input);
         if (area && (area->get_type() & DECORATION_AREA_MOVE_BIT))
         {
-            return {DECORATION_ACTION_MOVE, 0};
+            if (timer.is_connected()) /* we have a double-click (before release) */
+            {
+                return {DECORATION_ACTION_TOGGLE_MAXIMIZE, 0};
+            } else
+            {
+                timer.set_timeout(300, [=] ()
+                {
+                    timer.disconnect();
+                    return false;
+                });
+                // hardcoded 300ms, to be replaced by option in config:
+                // wf::option_wrapper_t<int> delay{"input/double_click_timeout"};
+                return {DECORATION_ACTION_MOVE, 0};
```

Comment:
I don't think this is very nice, we start a move action (potentially triggering wobbly & co) only to immediately exit it. Can we instead modify the code a bit more: trigger move on pointer motion if we have clicked on an area suitable for moving. (you'd also need to change the return type of the `handle_motion` method and handle it in deco-subsurface.cpp)

Reply:
agreed. (did not see as I don't use wobbly).
I just filled #1074 which is related but more complicated.